tools/converter/binary_reader/ReadManager.h

```diff
@@ -0,0 +1,723 @@
+#ifndef _ReadManager__
+#define _ReadManager__
+
```

Comment:
By the way, if these reader files are based on some other implementation, it might be helpful to add a reference to the repo? (might help in cross checking)

Reply:
What do you mean reference? submodule?

Reply:
no no..just comment / doc string